### DISCUSSION TITLE
Dynamically select issue labels

### DISCUSSION THREAD

**Discussion by mrousavy** [community/community#4651] [Projects and Issues] [answered]
It would be awesome to dynamically assign people and select issue labels depending on user choices.

For example, if there is a dropdown which selects the Platform:
![Screenshot 2021-07-14 at 11 16 05](https://user-images.githubusercontent.com/15199031/125596884-aa356aa1-a80d-478a-addb-55e30ac999ec.png)

and the user selects `iOS` I want to apply the `iOS` label to the issue. This could potentially also apply to assigning different people depending on the user's selection.

**Comment by lukehefson** [comment, marked as answer]
Please see: https://github.com/github/feedback/discussions/4266 ❤️
